### ISSUE TITLE
Accessing to external endpoint from the enclave having TLS issue.

### ISSUE THREAD

**Issue by ThantZin-develope** [aws-samples/aws-nitro-enclaves-workshop#46]
I followed the instruction as described in the secure local channel. But server application reside in the enclave environment always get TLS issue of tls:protocol not supported.

**Comment by xparticle** [comment]
what is the external endpoint you are trying to access from the enclave?

**Comment by ThantZin-develope** [comment]
https://dynamodb.ap-northeast-1.amazonaws.com

**Comment by xparticle** [comment]
You have to combine the functionality/instructions of traffic forwarder "secure local channel" and the way credentials are passed to the enclave in "cryptographic attestation" section to achieve connectivity with DynamoDB. And beyond connectivity if you also want to leverage attestation with decrypting values retrieved from DynamoDB, then further instructions from "cryptographic attestation" section around KMS integration have to be followed as well.

**Comment by ThantZin-develope** [comment]
Issue was fixed. As I said, I migrated the python codes into go lang. Python, forwarding the data to the destinated socket just done simply by passing the string. In golang, I gotta specify the byte size and that byte will be forwarded to the destinated socket. The issue was specfying the byte size. it should be like 1024 ,2048 etc etc. But I set 3000 ( non-sense ). Anyway, thanks you ( all of this workshop contributers ) so much. This tutorial helps me alot at learning nitro enclave.
